public/javascript/index.js

```diff
@@ -1,23 +1,121 @@
-const charactersAPI = new APIHandler('http://localhost:8000');
+const charactersAPI = new APIHandler('https://minions-api.herokuapp.com');
 
 window.addEventListener('load', () => {
   document.getElementById('fetch-all').addEventListener('click', function (event) {
+    charactersAPI
+      .getFullList()
+      .then(getData => {
+        ID = ''
 
+        getData.data.forEach(elm => ID += ` <div class="character-info">
+        <p>ID:${elm.id}</p>
+        <p>name: ${elm.name}</p>
+        <p>occupation:${elm.occupation}</p>
+        <p>cartoon:${elm.cartoon}</p>
+        <p>weapon:${elm.weapon}</p>
+        </div>`)
+
+        document.querySelector('.chartID').innerHTML = ID
```

Comment:
```suggestion
        document.querySelector('.characters-container').innerHTML = ID
```
Podrías seleccionar ese contenedor de manera que los recuadros se mantengan en el espacio indicado.